src/sonic-yang-mgmt/sonic_yang_ext.py

```diff
@@ -82,6 +82,10 @@ def _createDBTableToModuleMap(self):
         for j in self.yJson:
             # get module name
             moduleName = j['module']['@name']
+            # Skip sonic-types and sonic-extensions modules
+            if moduleName.strip() == 'sonic-types' or moduleName.strip() == 'sonic-extension':
```

Comment:
These yang models should not have top-level container, so Line 95 is good enough right?

Reply:
@praveen-li This change is copied from the [PR](https://github.com/Azure/sonic-buildimage/pull/6861/files) to avoid the build errors.

// Defined as container because it will be used as xpath in MUST/WHEN
    // expressions in yang.
    container operation {
        leaf operation {
            type enumeration {
                enum CREATE;
                enum UPDATE;
                enum DELETE;
            }
        }
        description "Sonic data operation types required during validation";
    }